cmd/operator-ui/actions/blackducks.go

```diff
@@ -82,7 +90,11 @@ func (v BlackducksResource) New(c buffalo.Context) error {
 	blackduckSpec := util.GetBlackDuckDefaultPersistentStorageLatest()
 	blackduck := &blackduckapi.Blackduck{}
 	blackduck.Spec = *blackduckSpec
-	blackduck.Spec.Namespace = ""
+	if v.config.IsClusterScoped {
+		blackduck.Spec.Namespace = ""
```

Comment:
Instead of "" here can we use the kube metava1.NamespaceAll?

Reply:
It's not default here.. It's for setting empty...

Reply:
True, but it is used as a filter to GetNamespace.  It so happens that "" is the same as NamespaceAll, but if "" were to no longer mean all namespaces on the GetNamespace functions then we would break.